### ISSUE TITLE
[Enhance] : Try to add Social links logo that would attract the users and could navigate to their desired page easily

### ISSUE THREAD

**Issue by DevBobby** [plone/volto#4314]
Adding social links to our website is a necessary step in order to increase visibility and engagement with our audience.

By including social media links on our website, we can increase the potential for our content to be shared, and to reach a larger audience. This can help to increase the visibility and credibility of our brand, and improve our search engine rankings.

Furthermore, adding social links to our website also allows our audience to easily follow us on social media platforms, which can help to increase our following, and in turn, increase our reach. This will also help in increasing our brand awareness and engagement with the audience.

In summary, including social media links on our website is a critical aspect of modern digital marketing and communication. It can help increase visibility, engagement, and reach, and is a necessary step in building a strong online presence for our brand.

**Comment by fredvd** [comment]
Hello @DevBobby  Your issue seems a lot like automated spam and AI generated text as it has nothing to do with the Volto project.   Please response to this message to clarify what it has to do with Volto or remove your issue.

Without any feedback  we'll remove your issue in due time and request a block for your account on the Plone and collective orgs on github.

**Comment by DevBobby** [comment]
@fredvd I'm sorry fred I was trying to put my words into more formal state so everyone would understand it. I do understand that i opened an issue related to frontend for plone webpage i was trying to suggest some features for the homepage.

**Comment by fredvd** [comment]
but for which homeoge do you want to add social link (buttons)l your personal volto project?  the https://plone.org homepage? the website of your organisation?

what did you already try? did you install Plone6 with the Volto frontend locally? did you search for addons?

**Comment by DevBobby** [comment]
I was talking about https://demo.plone.org/ the one the team is working on.

**Comment by stevepiercy** [comment]
LOL, I thought it was spam, too. Now that I understand what @DevBobby is talking about, this is so obvious a change, I completely missed it. Of course we should add Plone's social links to each demo site! This is a great suggestion.

But the correct repos for this issue are other ones, so this issue should be copied to those repos for each of the demo sites.

- https://demo.plone.org/
- https://6-classic.demo.plone.org/en
- https://52.demo.plone.org/

Additionally, I can never remember which repo is the source for each of the demo sites, so the GitHub octocat icon should be one of those social media links to each demo's repo.

**Comment by fredvd** [comment]
Aaaaah. Demo.plone.org (and the ClassicUI one).  

@DevBobby is it maybe also that demo.plone.org is shown on  the github repo homepage in the top right column? 

@stevepiercy There’s an underlying technical challenge we have to solve as well for both demo sites to be more atractive for ‘prospects’.  Demo misses default content on the site that shows off the hero and grid block and forms module.  The classicui site should at least have mosaic and maybe easyforms installed and presented with default content. 

For that we need to look at a suitable format to preload the sites with that more complex content structures. 

Installing a social links addon that has the links set to our other Plone sites can also be part of that.

This is also one of the priorities for the marketing team I think, but it’s all dependent on developer time, which we’re rather short on with so many people in the same teams.

We need to get more ‘latent’ or dormant developers in the community activated on tasks like these.  Which needs Project Management and precise and agreed on descriptions of what we want, so new devs can pick up tasks with confidence and ease. 

if we want to use the influx of gsoc students we need to have a backlog and todo list ready, once they pass the rite of installation…

cc:ing @ericof @pbauer @sneridagh @rioksane 

@ericof

**Comment by DevBobby** [comment]
@fredvd Thanks for sparing your time by looking at this issue. I see that the site uses pastanaga ui as a theme which is not quite attracting from a user perspective. As i suggest you guys can add this as a project and make the gsoc contributors work on it.Its from my perspective if you are participating in the gsoc this year.

**Comment by saadabban76** [comment]
Hey @fredvd, I would love to contribute to the "https://6-classic.demo.plone.org/en" site and make the website more user interactive by making the hero section look clean with good CSS and also add social links in the footer as @DevBobby suggested.

If you can send the demo website repo, I can start working on it right away :)

**Comment by Palakkgoyal** [comment]
Hey, @saadabban76 its been a month. Now I think that you are busy in something else so can I take this issue to solve.

**Comment by saadabban76** [comment]
Hi @Palakkgoyal, this issue has not been assigned to anyone for now.

**Comment by Palakkgoyal** [comment]
Okay

**Comment by stevepiercy** [comment]
@fredvd I think the first step is to create new issues for all the things that the Plone Marketing team wants to do.

As far as this issue only, what actually needs to be done?

Do we really have to wait for the Marketing Team to organize? For now I have labeled this issue as `26 status: blocked`.
